cluster/examples/kubernetes/cockroachdb/cluster.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-cockroachdb
+---
+apiVersion: cockroachdb.rook.io/v1alpha1
+kind: Cluster
+metadata:
+  name: rook-cockroachdb
+  namespace: rook-cockroachdb
+spec:
+  scope:
```

Comment:
need to add comments to the example yaml?

Reply:
the ceph cluster.yaml got really cluttered really quickly with comments.  I wonder if a better approach here would be to add a comment that points to the CRD documentation? https://github.com/jbw976/rook/blob/cockroachdb/Documentation/cockroachdb-cluster-crd.md